data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1853 @@
+
+event "resistance training: shaula empty"
+	system "Shaula"
+		remove fleet "Small Southern Pirates"
+		remove fleet "Large Southern Pirates"
+		remove fleet "Small Core Pirates"
+		remove fleet "Large Core Pirates"
+		remove fleet "Small Northern Pirates"
+		remove fleet "Large Northern Pirates"
+		remove fleet "Large Militia"
+		remove fleet "Large Free Worlds"
+
+event "resistance training: shaula back to normal"
+	system "Shaula"
+		add fleet "Small Southern Pirates" 500
+		add fleet "Large Southern Pirates" 800
+		add fleet "Small Core Pirates" 4000
+		add fleet "Large Core Pirates" 7000
+		add fleet "Small Northern Pirates" 5000
+		add fleet "Large Northern Pirates" 9000
+		add fleet "Large Militia" 9000
+
+event "resistance training done"
+
+mission "Resistance: Combat Training 3"
+	name "Pick Up Pyakri"
+	description "Pyakri and the other 53 Lunarium members are done with their training in human space. Head to <planet>, where they are waiting to be picked up and brought back to the Coalition."
+	landing
+	source
+		near "Aldhibain" 1 100
+	destination "Glaze"
+	to offer
+		has "event: resistance training done"
+		not "Heliarch Investigation 2: active"
+		not "Heliarch License: done"
+	to fail
+		has "Heliarch Investigation 2: active"
+	on offer
+		conversation
+			`Your monitor shifts into a messaging interface as you're landing, and some text quickly pours in. "Captain, hello. Pyakri, it is. In the last few months, fought many human pirates, and managed to practice and hone fleet movements, we have. Return to the others in the Coalition, we now must, so need your help again, we will. Waiting on <planet>, we will be." The message ends and your monitor returns to normal. You set a marker on <destination> to head there later and pick up the Lunarium members.`
+				accept
+
+mission "Resistance: Combat Training 4"
+	name "Homecoming"
+	description "Bring the <bunks> Lunarium members back to the Coalition, to <destination>."
+	passengers 54
+	landing
+	blocked "You need <capacity> in order to take on the next mission. Return here when you have the required space free."
+	source "Glaze"
+	destination "Factory of Eblumab"
+	to offer
+		has "Resistance: Combat Training 3: done"
+		not "Heliarch Investigation 2: active"
+		not "Heliarch License: done"
+	to fail
+		has "Heliarch Investigation 2: active"
+	on offer
+		payment 3000000
+		conversation
+			`Pyakri's fleet lands on the pads around your ship, and they message you saying they will wait for nightfall to board your ship. Once the sun sets you notice them opening their hatches, so you open your own and help them quickly and quietly sneak into your ship. Perhaps because they'd grown used to human space, or maybe they simply did not care since they were going home for good, they proved a lot less cautious this time around, though you still don't imagine anyone saw them.`
```

Comment:
```suggestion
			`Pyakri's fleet lands on the pads around the <ship>, and they message you saying they will wait for nightfall to board your ship. Once the sun sets you notice them opening their hatches, so you open your own and help them quickly and quietly sneak into your ship. Perhaps because they'd grown used to human space, or maybe they simply did not care since they were going home for good, they proved a lot less cautious this time around, though you still don't imagine anyone saw them.`
```